app/models/student_loans.rb

```diff
@@ -1,6 +1,10 @@
 module StudentLoans
   def self.start_page_url
-    "https://www.gov.uk/guidance/teachers-claim-back-your-student-loan-repayments"
+    if Rails.env.production?
+      "https://www.gov.uk/guidance/teachers-claim-back-your-student-loan-repayments"
+    else
+      "student-loans/claim"
```

Comment:
```suggestion
      "/student-loans/claim"
```
Otherwise you'll get a `No route matches [GET] "/student-loans/student-loans/claim"` error when clicking the service link at the top of the page

Reply:
We could even do `"/#{self.routing_name}/claim"` to make it a little more dry

Reply:
Like it 👍